handlers/Services/index.js

```diff
@@ -39,7 +39,8 @@ class Services extends React.Component {
     this.setState({target}, _ => {
       // TODO: sync this with the breakpoints in ./styles.css
       if (typeof matchMedia === 'function' && matchMedia('screen and (max-width: 600px').matches) {
-        setTimeout(__ => this.refs[target].getDOMNode().scrollIntoView(true));
+        //setTimeout(__ => this.refs[target].getDOMNode().scrollIntoView(true));
+        window.location = '/services/' + target;
```

Comment:
looks like it was added because the mobile layout has changed and the ```li``` become a list of links. gonna leave this in for now, but will talk to mel about the overall design of it